client/shared/tsconfig.json

```
@@ -11,6 +11,6 @@
     "rootDir": ".",
     "outDir": "./out",
   },
-  "include": ["**/*", ".*", "./src/**/*.json"],
+  "include": ["**/*", ".*", "./src/**/*.json", "/client/common/**/*.ts"],
   "exclude": ["../../node_modules", "./node_modules", "./out", "./src/end-to-end"],
```

A reference should be added to every package that uses `@sourcegraph/common`. Would you please double-check other packages and update `tsconfig.json` accordingly?

```suggestion
  "exclude": ["../../node_modules", "./node_modules", "./out", "./src/end-to-end"],
  "references": [{ "path": "../common" }],
```